lte/gateway/python/magma/subscriberdb/processor.py

```diff
@@ -108,8 +109,13 @@ class Processor(GSMProcessor, LTEProcessor):
     """
 
     def __init__(
-            self, store, default_sub_profile,
-            sub_profiles, op=None, amf=None,
+        self,
+        store,
+        default_sub_profile,
+        sub_profiles,
+        op=None,
+        amf=None,
+        sub_network=CoreNetworkType(),
```

Comment:
we should avoid initializing class members with default arguments
```suggestion
        sub_network=None,
```

and on line 132 do:
`self._sub_network = sub_network or CoreNetworkType()`

Reply:
Modified as suggested.